### PR TITLE
Hotfix: Disable to allocate protected path for one switch flow by PATCH

### LINKED ISSUE #5084: Flow updated after failed PATCH request
**Steps to reproduce:**
1. Create one switch flow with 0 vlan (or vlans)
```
{
  "flow_id": "flow1",
  "source": {
    "inner_vlan_id": 0,
    "port_number": 10,
    "switch_id": "2",
    "vlan_id": 0
  },
  "destination": {
    "inner_vlan_id": 0,
    "port_number": 11,
    "switch_id": "2",
    "vlan_id": 123
  },
  "maximum_bandwidth": 10000
}
```
2. Patch the from with `allocate_protected_path: true` and stat vlans

API `PATCH /v2/flows/{flow_id}`
```
{
"allocate_protected_path": true,
"statistics": {
    "vlans": [
      123, 456
    ]
  }
}
```

**Expected result:**
1. Bad Request
2. Flow would **not** be updated

**Actual Result:**
1. Bad Request
2. Flow has updated stat vlans and status info with error

### PR DESCRIPTION
**Pull request by niksv**
Closes #5084

### CODE REVIEW

File: src-java/nbworker-topology/nbworker-storm-topology/src/test/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsServiceTest.java [hunk 1]
```diff
@@ -290,8 +306,98 @@ public void shouldNotUpdateVlanStatistics() throws FlowNotFoundException {
...
+
+    @Test
+    public void ableToMakeProtectedFlowFromMultiSwitchFlowTest() throws FlowNotFoundException {
+        createFlow(FLOW_ID_1, switchA, switchB, false);
+        FlowPatch receivedFlow = FlowPatch.builder()
+                .flowId(FLOW_ID_1)
+                .allocateProtectedPath(true)
+                .build();
+        flowOperationsService.updateFlow(new FlowCarrierImpl(), receivedFlow);
+        // no exception expected
```
Comment (dmitrii-beliakov) [inline src-java/nbworker-topology/nbworker-storm-topology/src/test/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsServiceTest.java:359]: assertj has an idiom for no exceptions thrown. There is also a way to tell the same using junit annotation `@Test(expected = None.class)`. The difference is the test fails and handled properly or the test execution crashes.

**Review by dmitrii-beliakov — APPROVED** [review]

File: src-java/nbworker-topology/nbworker-storm-topology/src/test/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsServiceTest.java (hunk 1 above)
Comment (niksv) [reply src-java/nbworker-topology/nbworker-storm-topology/src/test/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsServiceTest.java:359]: I tried to do it, but Idea marked it as warning because (expected = None.class) is default value

**Comment by niksv** [comment]
Will be tested as part of RC
